core/com.b2international.snowowl.api.rest/META-INF/MANIFEST.MF

```diff
@@ -52,7 +52,8 @@ Require-Bundle: com.google.guava;bundle-version="[18.0.0,19.0.0)",
  com.fasterxml.jackson.module.jackson.module.scala;bundle-version="2.8.10",
  org.scala-lang.scala-library;bundle-version="2.10.6",
  com.fasterxml.jackson.module.jackson-module-paranamer;bundle-version="2.8.10",
- com.thoughtworks.paranamer;bundle-version="2.8.0"
+ com.thoughtworks.paranamer;bundle-version="2.8.0",
+ com.b2international.snowowl.datastore
```

Comment:
If you add a Require-Bundle header, then remove all corresponding Import-Package headers from the imported bundle.

Reply:
Done!